### PR TITLE
Fixes Issue #283

### PR DESCRIPTION
**Pull request by cool-ant**

### CODE REVIEW

File: packages/webapp/src/_app/ui/card.tsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ interface Props {
 }
 
 export const Card = ({ children, title, titleSize = 1, className }: Props) => (
-    <div className={`bg-white mb-5 p-6 md:p-8 w-full ${className}`}>
+    <div className={`bg-white mb-5 p-6 md:p-8 w-full ${className || ""}`}>
```
Comment (cool-ant) [inline packages/webapp/src/_app/ui/card.tsx:11]: This was showing "undefined" in the rendered HTML.

**Review by brandonfancher — CHANGES_REQUESTED** [review]
Here's some feedback after a precursory UI review. Diving into code next.

image

**Comment by brandonfancher** [comment]
@cool-ant When it comes to the spacing of the contents of the "Welcome to Eden" card, easiest thing to do would likely be to revert that card back to the state it was in prior to the addition of the stats. Try copying what's here and using that: https://github.com/eoscommunity/Eden/blob/77cd1f627f292a2356984d8d065d1846674bbffe/packages/webapp/src/pages/index.tsx

That's going to do a couple things. It will move "Welcome to Eden" into the flow of the text (instead of it being a Card title) which will make the height of the text in the right column line up better with the height of the text in the left column (when there are two columns) and it will fix the inter-paragraph spacing issues.

**Comment by brandonfancher** [comment]
We need to remove the "See Pending Invitations to Eden" link from this page too:
image

**Comment by brandonfancher** [comment]
You'll want to install the "Prettier - Code formatter" extension in VSCode and make sure it's reformatting your code when you save the file. I recommend installing that and going back and re-saving any files you modified.

image

**Comment by brandonfancher** [comment]
I like the link on the "Membership." Can we make it gray? Something like this?
And let's change the text to say "See all pending community invitations".

image

```javascript
{isCommunityActive && (
    <Link
        href="/induction/pending-invitations"
        className="block w-full my-8 text-center"
    >
        <span className="text-gray-400">
            See all pending community invitations
        </span>
    </Link>
)}
```

File: packages/webapp/src/pages/index.tsx [hunk 2]
```diff
@@ -12,6 +12,33 @@ import {
...
 } from "_app";
 
+interface CommunityStatusProps {
+    memberStats: any
+}
+
+const CommunityStatsCard = ({memberStats}: CommunityStatusProps) => {
+    return memberStats && (
+        <Card title="Community Stats" titleSize={1} className="h-full space-y-8 text-center p-8 lg:p-12 lg:py-24 xl:p-16 text-md lg:text-xl">
+            <p>
```
Comment (brandonfancher) [inline packages/webapp/src/pages/index.tsx:22]: I'd recommend removing the `title` prop here and using a `Heading` inside the `Card` component. Then use flexbox inside the card to center everything. Something like:

```javascript
<Card className="h-full flex flex-col justify-center items-center space-y-4 text-center text-lg lg:text-xl">
    <Heading size={2} className="mb-2">
        Community Stats
    </Heading>
    <Link href="/members" className="font-medium">
        {memberStats.active_members} active member
        {memberStats.active_members !== 1 && "s"}
    </Link>
    <Link
        href="/induction/pending-invitations"
        className="font-medium"
    >
        {memberStats.pending_members} pending invitation
        {memberStats.pending_members !== 1 && "s"}
    </Link>
</Card>
```

File: packages/webapp/src/_app/ui/card.tsx (hunk 1 above)
Comment (brandonfancher) [reply packages/webapp/src/_app/ui/card.tsx:11]: Yeah, good catch.

File: packages/webapp/src/pages/index.tsx [hunk 3]
```diff
@@ -40,59 +62,50 @@ export const Index = () => {
...
-                                target="_blank"
-                                isExternal
-                            >
-                                eden.eoscommunity.org
-                            </Link>
-                            .
-                        </Text>
-                    </div>
+                        </div>
+                        </div>
```
Comment (brandonfancher) [inline packages/webapp/src/pages/index.tsx:102]: The spacing/indentation's still messed up on this file. Is Prettier autoformatting on save for you?

File: packages/webapp/src/pages/index.tsx [hunk 4]
```diff
@@ -12,6 +12,28 @@ import {
...
+                {memberStats.active_members !== 1 && "s"}
+            </Link>
+            <Link href="/induction/pending-invitations" className="font-medium">
+                {memberStats.pending_members} pending
+                    invitation
+                {memberStats.pending_members !== 1 && "s"}
+            </Link>
+        </Card>
+)}
+
```
Comment (brandonfancher) [inline packages/webapp/src/pages/index.tsx:36]: Let's move this component down below the main component. Sparky and I have agreed that the first component in a file should be the main one getting exported.

**Review by brandonfancher — APPROVED** [review]
Approved pending final requested changes included here and via DM on Telegram.

**Pull request merged**